perception/traffic_light_recognition/traffic_light_classifier/src/nodelet.cpp

```diff
@@ -18,55 +18,65 @@
 
 namespace traffic_light
 {
-void TrafficLightClassifierNodelet::onInit()
+TrafficLightClassifierNodelet::TrafficLightClassifierNodelet(const rclcpp::NodeOptions & options)
+: Node("traffic_light_classifier_node", options)
 {
-  nh_ = getNodeHandle();
-  pnh_ = getPrivateNodeHandle();
-  image_transport_.reset(new image_transport::ImageTransport(pnh_));
-  pnh_.param<bool>("approximate_sync", is_approximate_sync_, false);
+  using std::placeholders::_1;
+  using std::placeholders::_2;
+  is_approximate_sync_ = this->declare_parameter("approximate_sync", false);
   if (is_approximate_sync_) {
     approximate_sync_.reset(new ApproximateSync(ApproximateSyncPolicy(10), image_sub_, roi_sub_));
     approximate_sync_->registerCallback(
-      boost::bind(&TrafficLightClassifierNodelet::imageRoiCallback, this, _1, _2));
+      std::bind(&TrafficLightClassifierNodelet::imageRoiCallback, this, _1, _2));
   } else {
     sync_.reset(new Sync(SyncPolicy(10), image_sub_, roi_sub_));
     sync_->registerCallback(
-      boost::bind(&TrafficLightClassifierNodelet::imageRoiCallback, this, _1, _2));
+      std::bind(&TrafficLightClassifierNodelet::imageRoiCallback, this, _1, _2));
   }
-  ros::SubscriberStatusCallback connect_cb = boost::bind(&TrafficLightClassifierNodelet::connectCb, this);
```

Comment:
I couldn't find a replacement for SubscriberStatusCallback for publishers, so I modified to use timer instead.

Reply:
Makes sense.